docs/CHANGELOG.rst

```diff
@@ -33,16 +33,36 @@ Added
   - ``Picard Tools (v2.18.5)``
   - ``bedGraphToBigWig (kent-v365)``
 - Support filtering by type on feature API endpoint
+- Add BigWig output field to following processes: ``alignment-bowtie``,
```

Comment:
It would be cleaner to use a list notation with every process is in it's own line.